we.go

```diff
@@ -482,6 +486,8 @@ func printDay(w weather) (ret []string) {
 }
 
 func init() {
+	flag.IntVar(&days, "days", 1, "Number of days of weather data to be displayed. This should be between 0 to 5")
```

Comment:
Please remove the "0 to 5" sentence, since this restriction is backend specific and might change without notice.